dc/core/resources/BorderRectangle.qml

```diff
@@ -39,27 +38,9 @@ Item {
         width: Style.borderWidth
         height: Style.borderWidth
         radius: Style.borderWidth
-        color: Style.inactiveBorderColor
         anchors.horizontalCenter: parent.horizontalCenter
         anchors.verticalCenter: parent.verticalCenter
+        visible: contentwindow.border === borderRect.border || contentwindow.border === ContentWindow.NOBORDER
+        color: contentwindow.border === borderRect.border ? Style.activeBorderColor : Style.inactiveBorderColor
```

Comment:
there is twice the same comparison, maybe introcuce an isActive bool property for more readability?